### PR TITLE
Added drag-n-drop for cards

### PR DESCRIPTION
**Pull request by dmitryRomankov**

### CODE REVIEW

File: client/src/app/components/active-card/active-card.component.ts [hunk 1]
```diff
@@ -7,9 +7,11 @@ import { Component, OnInit } from '@angular/core';
 })
 
 export class ActiveCardComponent implements OnInit {
+  @Input() public myActiveCards;
```
Comment (vharadkou) [inline client/src/app/components/active-card/active-card.component.ts]: typings

File: client/src/app/components/player-hand/player-hand.component.ts [hunk 2]
```diff
@@ -9,11 +7,10 @@ import { CardsFacade } from 'store';
 })
 
 export class PlayerHandComponent implements OnInit {
-  public allCards$ = this.cardsFacade.allCards$;
+  @Input() public allCardsMy;
```
Comment (vharadkou) [inline client/src/app/components/player-hand/player-hand.component.ts]: typings

File: client/src/app/pages/fight/fight.component.ts [hunk 3]
```diff
@@ -12,10 +11,23 @@ import { State } from 'store';
...
+  public myActiveCards: Card[] = [];
+  public enemyActiveCards: Card[] = [];
 
-  constructor(private store: Store<State>) { }
+  constructor(private cardsFacade: CardsFacade) {
+    this.cardsFacade.loadCards();
+    this.cardsFacade.allCards$
+      .subscribe(card => {
+        this.allCardsMy = card;
+        this.allCardsEnemy = card;
```
Comment (vharadkou) [inline client/src/app/pages/fight/fight.component.ts]: ??

Comment (vharadkou) [reply client/src/app/pages/fight/fight.component.ts]: should be streams myCards$ enemyCards$ and async pipe in html

**Comment by dmitryRomankov** [comment]
[add drag-n-drop logic](https://trello.com/c/sgphmue8/21-add-drag-n-drop-logic)

**Pull request merged**